cluster/manifests/z-kubenurse/serviceaccount.yaml

```diff
@@ -0,0 +1,12 @@
+{{ if eq .Cluster.ConfigItems.network_monitoring_enabled "true" }}
```

Comment:
Does it need to be at the very end of the manifest list? Or can we rename the folder to `kubenurse` to avoid confusion?